examples/kubernetes/access_points/README.md

```diff
@@ -6,8 +6,7 @@ In this case, the separation is managed on the EFS side rather than the kubernet
 
 ### Create Access Points (in EFS)
 Following [this doc](https://docs.aws.amazon.com/efs/latest/ug/create-access-point.html), create a separate access point for each independent data store you wish to expose in your cluster, tailoring the ownership and permissions as desired.
-
-**Note**: Until [issue #167](https://github.com/kubernetes-sigs/aws-efs-csi-driver/issues/167) is resolved, you must use a separate file system for each access point if they are to be mounted in the same pod.
+There is no need to use different EFS volumes.
```

Comment:
this sentence seems out of place now, i would remove

Reply:
Well, this is actually the crux of the change. Previously, if you wanted multiple shared stores in the same pod, you *had* to use separate volumes; and if you didn't, things broke in mysterious ways. Now this is possible, and it may be quite desirable for consumers to avoid the extra overhead.

Is there some way I can rephrase to make that clearer, or somewhere else I can put it so the context is better?